src/enums/routeNames.ts

```diff
@@ -25,4 +25,8 @@ export enum RouteNames {
   REGISTRATION_DEFINE_BUSINESS = 'registration-define-business',
   REGISTRATION_PEOPLE_ROLES = 'registration-people-roles',
   REGISTRATION_REVIEW_CONFIRM = 'registration-review-confirm',
+
+  // Entity Dissolution
+  DISSOLUTION_FIRM= 'dissolution-firm',
```

Comment:
Please update the comment.

The name still clashes with the "other" dissolution filings, but since we don't know how this will change in the future, I think the name is OK.

Also please add a space before the `=`.